SWADroid/src/main/res/layout/button_list.xml

```diff
@@ -0,0 +1,213 @@
+            android:textOff='a'
+            android:textOn='a'
+            android:textSize="30dp"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button2"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonb"
+            android:checked="false"
+            android:text='b'
+            android:textAllCaps="false"
+            android:textColor="#ffffff"
+            android:textOff='b'
+            android:textOn='b'
+            android:textSize="30dp"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button3"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonc"
+            android:textAllCaps="false"
+            android:text='c'
+            android:textOn='c'
+            android:textOff='c'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button4"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttond"
+            android:textAllCaps="false"
+            android:text='d'
+            android:textOn='d'
+            android:textOff='d'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button5"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttone"
+            android:textAllCaps="false"
+            android:text='e'
+            android:textOn='e'
+            android:textOff='e'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button6"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_margin="5dp"
+            android:layout_weight="2"
+            android:background="@drawable/togglebuttonf"
+            android:textAllCaps="false"
+            android:text='f'
+            android:textOn='f'
+            android:textOff='f'
+            android:textSize="30dp"
+            android:textColor="#ffffff"
+            android:visibility="gone" />
+
+        <ToggleButton
+            android:id="@+id/button7"

```

Comment:
You should use a more descriptive name for the id. This improves the readability of the code that refers to this id and makes it easier to identify which button is being used.